tmol/score/bonded_atom.py

```diff
@@ -133,4 +133,6 @@ def bonded_path_length(
         cache=True,
     )
 
-    return csgraph.dijkstra(bond_graph, directed=False, unweighted=True, limit=limit)
+    return csgraph.dijkstra(
+        bond_graph.tocsr(), directed=False, unweighted=True, limit=limit
```

Comment:
Does this call to bond_graph.tocsr() produce a faster traversal?

Reply:
I think this conversion is happening automatically, but newer versions of sparse don't do this autoconversion so we make it explicit.  I actually have a branch where this dependency is removed, but didn't want to duplicate it here.